Gemfile

```diff
@@ -65,6 +65,7 @@ gem "optimist",                       "~>3.0",         :require => false
 gem "pg",                                              :require => false
 gem "pg-dsn_parser",                  "~>0.1.0",       :require => false
 gem "query_relation",                 "~>0.1.0",       :require => false
+gem "rack",                           "~>2.0.9",       :require => false
```

Comment:
We don't normally add "indirect" dependencies here so it might be worthwhile to say in a comment why we're doing it here.